tests/unit/PartialUserFormVerifyControllerTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Firesphere\PartialUserforms\Tests;
+
+use Firesphere\PartialUserforms\Controllers\PartialSubmissionController;
+use Firesphere\PartialUserforms\Controllers\PartialUserFormVerifyController;
+use Firesphere\PartialUserforms\Forms\PasswordForm;
+use Firesphere\PartialUserforms\Models\PartialFormSubmission;
+use SilverStripe\Control\NullHTTPRequest;
+use SilverStripe\Control\Session;
+use SilverStripe\Dev\SapphireTest;
+
+class PartialUserFormVerifyControllerTest extends SapphireTest
```

Comment:
Maybe this test should be a functional test, I see a lot of requests and sessions involved.

Reply:
🤷‍♂ There's a lot of confusion around what is what already when it comes to test naming, I can't be bothered that much about it to be honest.

Reply:
It's not about test naming, it's about extending the right class to be able to use the appropriate test functions for testing controllers. 
For example, testing controllers and forms should extend from FunctionalTest so functions such as `get`, `post`, or `submitForm` can be used. In this test, you could have just used the post/get functions and not create controllers, session and requests manually. It's just my thoughts, you don't have to change it if you don't feel like it.

Reply:
I might refactor it later indeed, but for now, time-wise speaking, I'd rather leave it as-is